### PR TITLE
first step to a simple InitializedOperator

### PR DESCRIPTION
**Pull request by jdroenner**
remove params from InitializedOperatorImpl

### CODE REVIEW

**Comment by jdroenner** [comment]
[Refactor InitializedOperatorImpl](https://trello.com/c/ZgQvJKhw/177-refactor-initializedoperatorimpl)

File: operators/src/processing/point_in_polygon.rs [hunk 1]
```diff
@@ -77,7 +76,7 @@ impl VectorOperator for PointInPolygonFilter {
     }
 }
 
-pub type InitializedPointInPolygonFilter = InitializedOperatorImpl<(), VectorResultDescriptor, ()>;
+pub type InitializedPointInPolygonFilter = InitializedOperatorImpl<VectorResultDescriptor, ()>;
```
Comment (ChristianBeilschmidt) [inline operators/src/processing/point_in_polygon.rs:79]: I guess we should use the moment to introduce an empty dummy struct here as state.

**Comment by ChristianBeilschmidt** [comment]
I guess that is an improvement :+1:. However, the task states that we tried to move from *Lambda function to builder pattern*. This is not in this PR, yet.

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]

**Comment by jdroenner** [comment]
i know but i had to start somewhere.

**Comment by jdroenner** [comment]
will add builder later

**Review by ChristianBeilschmidt — APPROVED** [review]

**Pull request merged**